packages/shadcn/src/AddButton/AddButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { FormContextType, IconButtonProps, RJSFSchema, StrictRJSFSchema, TranslatableString } from '@rjsf/utils';
+import { PlusCircle } from 'lucide-react';
+import { Button } from '../components/ui/button';
+
+export default function AddButton<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>({
```

Comment:
Can you add JSDocs to your components? Likely you can adapt what is present in other themes